src/alpaka/AlpakaDataFormats/TrackingRecHit2DAlpaka.h

```diff
@@ -73,6 +73,11 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE {
       // SoA view on device:
       auto view_h{cms::alpakatools::createHostView<TrackingRecHit2DSOAView>(&view, 1u)};
       alpaka::memcpy(queue, m_view, view_h, 1u);
+#ifdef ALPAKA_ACC_CPU_B_TBB_T_SEQ_ENABLED
```

Comment:
For some reason, if `alpaka::wait(queue)` is inside this `#ifdef`, it seems like it has no effect, and the problem remains